tools/deserialize-proto/main.go

```diff
@@ -141,13 +143,17 @@ func printProtoMessagesFromStdin(in io.Reader, out io.Writer, msg proto.Message)
 			return fmt.Errorf("%w: %w", errUnmarshal, err)
 		}
 
-		m := jsonpb.Marshaler{Indent: "  ", EmitDefaults: true}
-		json, err := m.MarshalToString(msg)
+		m := protojson.MarshalOptions{EmitUnpopulated: true, EmitDefaultValues: true}
```

Comment:
It might be worth adding options to the `jsonutil` functions (`WithEmitDefaults` that would add these two options to the object created down in `marshalToString`).

Reply:
I thought about it but this is the only place where we need it and it's a tool so I think we should keep it simple